src/backend/api/Fusion.Resources.Api/Controllers/PersonAbsence/ApiModels/ApiPersonAbsence.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Fusion.Resources.Domain;
+using System;
+using System.Text.Json.Serialization;
+
+namespace Fusion.Resources.Api.Controllers
+{
+    public class ApiPersonAbsence
+    {
+        public ApiPersonAbsence(QueryPersonAbsence absence)
+        {
+            Id = absence.Id;
+            Created = absence.Created;
+            CreatedBy = new ApiPerson(absence.CreatedBy);
+            Comment = absence.Comment;
+            AppliesFrom = absence.AppliesFrom;
+            AppliesTo = absence.AppliesTo;
+            Type = Enum.Parse<ApiAbsenceType>($"{absence.Type}", true);
+
+            // If null, absence percentage should be displayed as 100
+            AbsencePercentage = absence.AbsencePercentage ?? 100;
```

Comment:
I think this should be nullable as well... I don't think a percentage is fitting in all types.. 
Ex. you will never be on 70% vacation. 

If you are on 100% on vacation, you can still have a 100% position in a project, that does not mean you work 200% that period... 
So it makes more sense that it is null, just to indicate a situation...